identity-apps-core/apps/authentication-portal/src/main/webapp/fido2-enrol.jsp

```diff
@@ -0,0 +1,417 @@
+                <jsp:include page="extensions/product-title.jsp"/>
+            <% } else { %>
+                <jsp:include page="includes/product-title.jsp"/>
+            <% } %>
+        </layout:component>
+        <layout:component componentName="MainSection" >
+            <div class="ui segment">
+                <div id="loader-bar" class="loader-bar"></div>
+
+                <h3 class="ui header">
+                    <span id="fido-header">
+                        <%=AuthenticationEndpointUtil.i18n(resourceBundle, "fido.title.registration.instruction")%>
+                    </span>
+                    <span id="fido-keyname-header" style="display: none;">
+                        <%=AuthenticationEndpointUtil.i18n(resourceBundle, "fido.title.name.passkey")%>
+                    </span>
+                    <span id="fido-header-error" style="display: none;">
+                        <%=AuthenticationEndpointUtil.i18n(resourceBundle, "fido.title.error.registration")%>
+                    </span>
+                </h3>
+                <div class="ui two column left aligned stackable grid">
+                    <div id="fido-initialize" class="middle aligned row">
+                        <div class="six wide column">
+                            <img class="img-responsive" src="images/U2F.png" />
+                        </div>
+                        <div class="ten wide column">
+                            <p id="general-browser-instruction">
+                                <%=AuthenticationEndpointUtil.i18n(resourceBundle, "fido.info.registration.instruction")%>
+                            </p>
+                            <div id="safari-instruction" style="display:none">
+                                <p>
+                                    <%=AuthenticationEndpointUtil.i18n(resourceBundle, "fido.failed.instruction" )%>
+                                </p>
+                                <div class="ui divider hidden"></div>
+                                <button class="ui primary fluid large button" id="initiateFlow" type="button" onclick="talkToDevice()">
+                                    <%=AuthenticationEndpointUtil.i18n(resourceBundle, "fido.proceed" )%>
+                                </button>
+                            </div>
+                        </div>
+                    </div>
+                    <div id="fido-keyname-content" style="display: none;" class="middle aligned row">
+                        <div class="sixteen wide column">
+                            <p>
+                                <%=AuthenticationEndpointUtil.i18n(resourceBundle, "fido.info.name.passkey" )%>
+                            </p>
+                            <div class="ui form">
+                                <!-- Add an element to display the error message -->
+                                <div class="ui red message" id="keynameError" style="display: none;"> Passkey name cannot be empty.</div>
```

Comment:
Shall we i18n this error message?

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/4156/commits/8968e9e9e80de6f997577d12f3f6222db508b1de